lib/Controller/DaysController.php

```diff
@@ -159,8 +159,7 @@ public function day(string $id): JSONResponse
 
     /**
      * @NoAdminRequired
-     *
-     * @PublicPage
+     * @NoCSRFRequired
```

Comment:
This endpoint doesn't change state, so this wouldn't make much difference. But this definitely isn't the right way to do this: the app should be sending the CSRF token instead. This might work as a quick fix for now but there'll be trouble with other endpoints which do change state (can't remove CSRF protection on those)

Reply:
OK, then. I'll find out how to fetch a Csrf token and sent it along